src/reducers/selectedIncidentReducer.jsx

```diff
@@ -32,6 +33,8 @@ const selectedIncidentReducer = (state = initialState.selectedIncident, action)
       };
     case ADD_CHAT:
       return { ...state, chats: [...state.chats, action.chat] };
+    case GET_SLACKCHATS:
+      return { ...state, chats: [...action.chats] };
```

Comment:
this overrides the chats that were loaded with the incident details.

Reply:
Thanks for your feedback @yasashiku, the api to fetch chats by incidences have not been implemented on the backend. The task was to ensure that, in the meantime, chats are been _fetched from the database_ as opposed to using _mock data_.